src/prompt.c

```diff
@@ -72,7 +77,7 @@ void	set_current_path(t_prompt *_prompt, const char *_path)
 
 void 	prompt_print(t_prompt *_prompt)
 {
-	printf("%s", _prompt->buffer);
+	dprintf(2, "%s", _prompt->buffer);
```

Comment:
use STDERR, as defined in libft.h

(never ever use a magical value like '2' in your code)